src/shared/firewall-util.c

```diff
@@ -13,12 +13,18 @@ enum backend {
 #ifdef HAVE_LIBIPTC
         FW_BACKEND_IPTABLES,
 #endif
+        FW_BACKEND_NFTABLES,
 };
 
 static enum backend fw_backend;
 
 static enum backend fw_backend_probe(void)
 {
+        int ret = fw_nftables_init();
+
+        if (ret == 0)
+               return FW_BACKEND_NFTABLES;
```

Comment:
why declare `int ret`? You could as well just write `if (fw_nftables_init() == 0)` here...